SU-HSDO/suhumsci
# shs-10: bugfix for single hero 'slideshows'

# READY FOR REVIEW

## Summary
This fixes a bug in the current release related to this PR: https://github.com/SU-HSDO/suhumsci/pull/1107. If the slideshow only has one slide, the text animation wouldn't fire. This fixes.

## Need Review By (Date)
_['asap']_

## Steps to Test
1. Go to a Flexible page and add a 'Hero (Gradient) Slider' to the Main Region content. Only add one slide. Save and make sure the text animates in.
2. Edit this page and add a couple more slides. Make sure each slide animates the text as it comes in.

## Screenshot
![Screen Shot 2022-05-24 at 9 37 28 AM](https://user-images.githubusercontent.com/8405274/170049575-2c4b33b3-93d0-4177-8453-5c4c9b87833a.png)
